docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,333 @@
+where the higher the number, the more optimized the result.
+You can find more information about this parameter
+`here <https://qiskit.org/documentation/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.html#Preset-Pass-Managers>`_.
+
+Initialize the quantum circuit
+==============================
+
+The effect of setting the :attr:`~qiskit.transpile.optimization_level` will differ depending on the backend you are using.
+For this example, you will explore the `CSWAP <https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.cswap.html>`_ gate,
+which is a three qubit gate which will be transpiled into one and two qubit gates.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister 
+    from qiskit.compiler import transpile
+    from qiskit.providers.fake_provider import FakeQuito
+
+    backend = FakeQuito()
+
+    qc = QuantumCircuit(3) # Initialize the quantum circuit with 3 qubits.
+    
+    qc.cswap(qr[0],qr[1],qr[2]) # Add the cswap gate to the quantum circuit.
+    
+    print(qc)
+
+.. testoutput::
+
+    q0_0: ─■─
+           │ 
+    q0_1: ─X─
+           │ 
+    q0_2: ─X─
+             
+Using backend’s information
+===========================
+
+
+You should adhere to the specific configuration of your backend when utilizing :meth:`~qiskit.transpile`. 
+This process entails breaking down your circuit into :attr:`~qiskit.transpile.basis_gates`` and considering the physical connections specified in the 
+:attr:`~qiskit.transpile.coupling_map` for two qubit gates.
+Given the presence of noise in the backend, it is crucial to optimize your circuit by adjusting the :attr:`~qiskit.transpile.optimization_level` parameter. 
+This will help minimize the number of circuit operations and enhance the overall performance.
+
+
+
+What each optimization level does
+---------------------------------
+
+
+When using a backend, you can access its properties through the instruction  :meth:`backend.configuration()`.
+These properties, such as basis gates, coupling map, and init layout, play a crucial role in shaping the behavior of the quantum circuit.
+
+For example, with :meth:`~qiskit.providers.fake_provider.FakeQuito`, you can learn about its qubit connections and the gates it uses to generate your quantum circuits.
+
+.. testcode::
+
+    print(f"Basis gates of your backend: {backend.configuration().basis_gates}")
+    print(f"Coupling map of your backend: ",{backend.configuration().coupling_map}")
+
+.. testoutput::
+
+    Basis gates of your backend:  ['id', 'rz', 'sx', 'x', 'cx', 'reset']
+    Coupling map of your backend:  [[0, 1], [1, 0], [1, 2], [1, 3], [2, 1], [3, 1], [3, 4], [4, 3]]
+
+When setting the ``optimization_level`` to 0, the resulting quantum circuit is not optimized and utilizes only three qubits. 
+The coupling map, represented by the subset **[[0,1],[1,0],[1,2],[2,1]]**, indicates the physical qubits available in the backend. 
```

Comment:
This is a personal preference, but wouldn't it be nicer to wrap the coupling map and basis gates in code-font, as these are already in Python notation? (Same for the basis gates below)
```suggestion
The coupling map, represented by the subset ``[[0,1],[1,0],[1,2],[2,1]]``, indicates the physical qubits available in the backend. 
```

Reply:
I was thiking the same